onnxruntime/test/testdata/kernel_def_hashes/contrib.cpu.json

```diff
@@ -181,15 +185,23 @@
     ],
     [
         "QLinearAveragePool com.microsoft CPUExecutionProvider",
-        9152647959212466896
+        7697522567125769728
```

Comment:
for QLinearAveragePool and QLinearGlobalAveragePool:
a change in the kernel def hash will break older models that use these ops. so we are going from a single kernel (that only handled uint8_t before) to one kernel each for uint8_t and int8_t.
I'd suggest keeping a single untyped kernel registration and handling both types in its kernel implementation. it would also be possible to further limit the amount of templated code within the kernel implementation which typically helps with binary size. at the moment the whole kernel class is templated.
it would also be possible to use KernelDefBuilder::FixedTypeConstraintForHash() to preserve the original hash for the uint8_t kernel and add a new one for int8_t, but I'd recommend the former approach if possible.